autoapi/mappers/python.py

```diff
@@ -19,8 +18,7 @@ def read_file(self, path, **kwargs):
 
         :param path: Path of file to read
         '''
-        # TODO support JSON here
-        # TODO sphinx way of reporting errors in logs?
+        from epyparse import parsed
```

Comment:
A more expected pattern here would be fencing the import in the module scope with a `sys.version_info < (3,)` check